chart/queue-worker/README.md

```diff
@@ -0,0 +1,14 @@
+# JetStream for OpenFaaS queue-worker
+
+Deploy OpenFaaS queues.
+
+## Install an additional queue
+You can run the following command from within the `faas-netes` folder, not the chart's folder.
```

Comment:
This chart needs to be part of `make charts` so that it can be published and installed without cloning the repo.

Keep the local instruction and mark it as for development.